docs-src/python/how-to-set-a-dynamic-signal-in-python.md

```diff
@@ -0,0 +1,31 @@
+---
+id: how-to-set-a-dynamic-signal-in-python
+title: How to set a Dynamic Signal
+sidebar_label: Set a Dynamic Signal
+description: Use `dynamic=True` on the `@workflow.signal` decorator to make a Signal dynamic.
+tags:
+- dynamic signal
+- python sdk
+- code sample
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-python/blob/dynamic-ent/dynamic_entities/your_dynamic_signal_dacx.py. -->
+
+A Dynamic Signal in Temporal is a Signal that is invoked dynamically at runtime if no other Signal with the same name is registered.
+A Signal can be made dynamic by adding `dynamic=True` to the `@signal.defn` decorator.
+
+The Signal Handler must then accept a single argument of type `Sequence[temporalio.common.RawValue]`.
```

Comment:
Same as query, this accepts a string name

Reply:
Fixed.